source/includes/_tipser-elements.md

```diff
@@ -40,12 +40,28 @@ Start the application
 `npm start`
 
 ### Example inserting elements in your site
-You can combine Tipser Elements with your own application
+You can combine Tipser Elements with your own application:
 
-- `TipserElementsProvider` entry point to Tipser Elements (creating a context for other Elements) with the `tipserElementsConfig` as props.
-- `TipserElement` is a generic Element that can render any Contentful content that's fed as a prop to the element.
-- `TipserProduct` is the Element that renders the product given the Tipser product ID as the prop.
-- `CartIcon` is the Element that displays the number of items in your cart and gives the user a way to open the checkout dialog.
+#### `TipserElementsProvider` 
+Entry point to Tipser Elements (creating a context for other Elements);
+
+prop name  | description | type  | required | default value 
+-----------|-------------|-------|----------|--------------
+posId | id of Point of sale | string | true | 
+config | configuration object | {} | false   | {}
+defaultAddedToCartPopup | controls whether default Added To Cart Popup is displayed | boolean | false | true
```

Comment:
This should be part of the `config` obect instead of a new props.

Reply:
I also thought about that, but it is easier to document props rather then properties of object.

What do you think about that ? Do you have any idea how I should in a clear way document properties of conf object ?

Reply:
@chmurson we would like to keep the number of props minimal here to not pollute the API of this component it too much (list of all the config options may be long). Hence, most of the configuration options should be bundled within `config` property. `posId` is the only exception as it's critical and strictly required. For example of how to document that object, see here: https://tipser.github.io/docs/#all-configuration-options .

Reply:
Yeah, I thought you suggest something like that. Okay I've tried it, and this how it looks:

<img width="1074" alt="Zrzut ekranu 2019-04-29 o 15 08 47" src="https://user-images.githubusercontent.com/3996881/56898372-2aa88480-6a91-11e9-948a-b634956acb8d.png">

Reply:
ok done!